docker-compose.yml

```diff
@@ -29,9 +27,9 @@ services:
       - source: quote_txt
         target: quote.txt
```

Comment:
As mentioned elsewhere, the `secrets` block can be removed if you are not using it.

Reply:
@dav3r resolved with [598d748](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/598d748b6771cf70e8fde93a7114349e7fab7e4f)